CHANGELOG.md

```diff
@@ -15,8 +15,9 @@ This file is used to list changes made in each version of the AWS ParallelCluste
 3.13.2
 ------
 
-**CHANGES**
-- Fix build image failures occurring on non-latest versions of Rocky Linux 9.
+**BUG FIXES**
+- Fix `build-image` failure on Rocky 9, occurring when the parent image does not ship the latest kernel version.
```

Comment:
Nit: missing period at the end of the sentence. Not a blocker of corse, but just to keep in mind.